playbooks/kvm-deploy_caasp4.yml

```diff
@@ -0,0 +1,45 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- hosts: localhost
+  gather_facts: yes
+  any_errors_fatal: true
+  vars_files:
+    - "{{ playbook_dir }}/../vars/common-vars.yml"
+    - "{{ playbook_dir }}/../vars/deploy-on-kvm.yml"
+  tasks:
+    - import_role:
+        name: caasp4
+        tasks_from: setup_terraform_workspace_libvirt
+      vars:
+        skuba_ci_terraform_worker_count: "{{ deploy_on_kvm_caasp_workers }}"
+        skuba_ci_terraform_workspace: "{{ socok8s_workspace }}/skuba-terraform/"
+
+    - import_role:
+        name: caasp4
+        tasks_from: setup_nodes
+      vars:
+        _import_terraform_resources: false
```

Comment:
This seems like it should be a normal var. The `_` seems to indicate that is temporary, butr no other declared var from playbooks (other that commmand outputs) is using this, so its a bit confusing to track it down.

Reply:
Removed the `_`. Marking as solved conversation for now.